docs/rackhd/event_notification.rst

```diff
@@ -1,105 +1,272 @@
 Event Notification
 ------------------
 
-The change of resources managed by RackHD could be retrieved from AMQP messages.
-
-- Exchange: **on.events**
-- Routing Key **event.<type>**
+RackHD support event notification via both AMQP and web hook.
 
-`<type>` is event's type, it's the same with event's payload `type`, see payloads_ for more details.
+A web hook allows an application to subscribe certain RackHD published events by configured URL, when one of the subscribed events is triggered, RackHD will send a HTTP POST request with event payload to configured URL.
```

Comment:
I think we need a section to introduce how the user design the hook service and URL, at least following should be covered:
- RackHD uses verb `POST` to send the event to web hook URL.
- The Content-Type is `applicaton/json`.
- The event data is stored in the request body.